### PR TITLE
Try py3.7

### PR DESCRIPTION
**Pull request by dneise**
This is the continuation of PR #751. See also the discussion there as to why current the TARGET plugin is not installed for some systems and some python versions.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -59,9 +59,9 @@ before_install:
 
 install:
     - conda create --name cta-dev python=$PYTHON_VERSION pip
-    - conda env update -n cta-dev --file environment.yml
+    - travis_wait 20 conda env update -n cta-dev --file environment.yml
     - source activate cta-dev
-    - ulimit -s 16000  # increase stack size limit, for libhessio
+    - ulimit -s 50000  # increase stack size limit, for libhessio 50MB
```
Comment (MaxNoe) [inline .travis.yml]: I think this is 50G:

```
$ ulimit -a | grep stack
stack size              (kbytes, -s) 8192
```

Comment (dneise) [reply .travis.yml]: Yep you are right ... thanks ...

Comment (dneise) [reply .travis.yml]: let's see if 50GB helps with this test

**Comment by dneise** [comment]
superseded by #925 - closed

**Pull request closed without merging**